frontend/src/lifetime/errors.ts

```diff
@@ -71,12 +71,11 @@ function githubUrl(panicDetails: string): string {
 		${browserVersion()}, ${operatingSystem()}
 
 		**Stack Trace**
-		Copied from the crash dialog in the Graphite Editor:
+		Copied from the crash dialog in the Graphite Editor:`;
```

Comment:
It's slightly stylistically preferable to put the closing backtick on the next line. I think that also means you can avoid the double `\n` on the next concatenation.

Reply:
You do trim.

Reply:
Gotcha, no worries then.